content/v2.0/reference/flux/stdlib/pagerduty/endpoint.md

```diff
@@ -49,6 +48,8 @@ The returned object must include the following fields:
 - `client`
 - `client_url`
 - `class`
+- `dedupKey`
```

Comment:
This is not true:

```
// Note that while sendEvent accepts a dedup key, endpoint gets the dedupkey from the groupkey of the input table instead of it being handled by the `mapFn`.
```